benchmarks/migrations/0051_manifestreduced_created_at.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+INIT_MANIFESTREDUCED_CREATED_AT = """
+UPDATE benchmarks_manifestreduced
+SET created_at = (
+  SELECT min(created_at)
+  FROM benchmarks_result
+  JOIN benchmarks_manifest ON benchmarks_manifest.id = benchmarks_result.manifest_id
+  WHERE benchmarks_manifest.reduced_id = benchmarks_manifestreduced.hash
+)
+"""
```

Comment:
Do I have to remove reduced manifests that point to nothing? Here is what I'm getting when trying this migration on production snapshot


```
   Applying benchmarks.0051_manifestreduced_created_at...Traceback (most recent call last):
   File "./manage.py", line 10, in <module>
     execute_from_command_line(sys.argv)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
     utility.execute()
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
     self.fetch_command(subcommand).run_from_argv(self.argv)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
     self.execute(*args, **cmd_options)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
     output = self.handle(*args, **options)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 222, in handle
     executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 110, in migrate
     self.apply_migration(states[migration], migration, fake=fake, fake_initial=fake_initial)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 148, in apply_migration
     state = migration.apply(state, schema_editor)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/migrations/migration.py", line 115, in apply
     operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/migrations/operations/special.py", line 102, in database_forwards
     self._run_sql(schema_editor, self.sql)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/migrations/operations/special.py", line 127, in _run_sql
     schema_editor.execute(statement, params=None)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/backends/base/schema.py", line 111, in execute
     cursor.execute(sql, params)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
     return super(CursorDebugWrapper, self).execute(sql, params)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
     return self.cursor.execute(sql, params)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
     six.reraise(dj_exc_type, dj_exc_value, traceback)
   File "/home/milosz/linaro/tcstorm/crayonbox-django/venv/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
     return self.cursor.execute(sql)
 django.db.utils.IntegrityError: null value in column "created_at" violates not-null constraint
 DETAIL:  Failing row contains (b7c0b6ef247754236337af8c5d1d1aeed5cfb844, null).

```